components/omega/doc/devGuide/CMakeBuild.md

```diff
@@ -34,7 +34,7 @@ with an underscore ("_").
 The following is a list of Omega-specific variables available in
 this version:
 
-```
+```vbnet
```

Comment:
@grnydawn, please take these `vbnet` tags out.  Sphinx doesn't seem to understand them.

Reply:
Other than this one issue, the docs build fine except for issues unrelated to this PR and addressed in #32.